framework/oryx-ml/src/main/java/com/cloudera/oryx/ml/MLUpdate.java

```diff
@@ -49,7 +49,7 @@
 import com.cloudera.oryx.common.collection.Pair;
```

Comment:
I think there are more renames needed, like, this is a file that needs to move to a subpackage as well